examples/maskrcnn-example/utils/encord_dataset.py

```diff
@@ -21,6 +21,16 @@ def __getitem__(self, idx):
 
         boxes, labels, area, iscrowd = [], [], [], []
         for target_item in target:
+            if (not target_item["bbox"][2] > 0) or (not target_item["bbox"][3] > 0):
+                print(
+                    f"ERROR: Target bbox for the following image \n"
+                    f'title:      {img_metadata[0]["image_title"]} \n'
+                    f'label_hash: {img_metadata[0]["label_hash"]} \n'
+                    f'data_hash:  {img_metadata[0]["data_hash"]} \n'
+                    f'has non-positive width/height => [x,y,w,h]: {target_item["bbox"]}. \n'
+                    f"Therefore, skipping this annotation."
+                )
+                continue
```

Comment:
This is also introduced in #89. We should rebase this commit on to of that branch.